### PR TITLE
Configuration of SpringBoot Application and basic dependencies

### LINKED ISSUE #1: Create a spring boot project with proper dependencies

### PR DESCRIPTION
**Pull request by Usha3012**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,12 +4,62 @@
...
   <dependencies>
-	
- </dependencies>
+      <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-web -->
+      <dependency>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-web</artifactId>
+          <version>2.2.4.RELEASE</version>
+      </dependency>
+      <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-data-jpa -->
```
Comment (njoysubho) [inline pom.xml]: remove this comments

File: pom.xml [hunk 2]
```diff
@@ -4,12 +4,62 @@
...
+      <dependency>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-data-jpa</artifactId>
+          <version>2.2.4.RELEASE</version>
+          <!--<scope>provided</scope> <scope>test</scope> -->
```
Comment (njoysubho) [inline pom.xml]: remove comments

File: pom.xml [hunk 3]
```diff
@@ -4,12 +4,62 @@
...
     <maven.compiler.source>${jdk.version}</maven.compiler.source>
 </properties>
   <dependencies>
-	
- </dependencies>
+      <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-web -->
+      <dependency>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-web</artifactId>
+          <version>2.2.4.RELEASE</version>
```
Comment (njoysubho) [inline pom.xml]: Don't think we need to mention version here spring boot starter parent will bring appropriate compatible version.
Can you try removing the versions and try compiling

File: README.md [hunk 4]
```diff
@@ -1 +1,12 @@
 # goal-service
+
+```text
```
Comment (njoysubho) [inline README.md:3]: No need to write TODOs here .
Rather add a section how to compile and Run the app

**Review by njoysubho — CHANGES_REQUESTED** [review]
@Usha3012 check my comments

File: README.md (hunk 4 above)
Comment (Usha3012) [reply README.md:3]: This is just for me to check the todo list , onces its done I will remove these.

File: pom.xml (hunk 3 above)
Comment (Usha3012) [reply pom.xml]: Without version , springboot starter parent not working

Comment (Usha3012) [reply pom.xml]: Done

File: pom.xml (hunk 1 above)
Comment (Usha3012) [reply pom.xml]: Done

File: pom.xml (hunk 2 above)
Comment (Usha3012) [reply pom.xml]: Done

**Review by njoysubho — APPROVED** [review]

**Pull request merged**